syntax = "proto3";
// The greeting service definition.
package hello;

// The request message containing the user's name.
message HelloRequest {
    string name = 1;
}

// The response message containing the greetings
message HelloResponse {
    string message = 1;
}

service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloResponse) {}
  // Sends another greeting
  rpc SayHelloAgain (HelloRequest) returns (HelloResponse) {}
}